### DISCUSSION TITLE
Where I can learn more about Ory?

### DISCUSSION THREAD

**Discussion by kuldeepstechwork** [ory/awesome-ory#46] [Q&A] [answered]
Hi, I have successfully cloned the repo and it is running at my PC. I am getting a signup and login page. I have created my user account and it is working. I am new to ory. Can you guys share resources where I can learn more about ory . I also want to learn what things I can do with ory. Please help.

**Comment by vinckr** [comment, marked as answer]
Hello @k8607981392 

What are you trying to achieve? If you want to learn more about Ory the documentation is a great place to start:

https://www.ory.sh/docs/welcome

This document is also a good overview: https://www.ory.sh/docs/ecosystem/projects

For Identity Management: https://www.ory.sh/docs/kratos/
For Permissions & Access Control: https://www.ory.sh/docs/keto/
For OAuth2 Server: https://www.ory.sh/docs/hydra/
For Zero Trust/Identity Aware Proxy: https://www.ory.sh/docs/oathkeeper/

Also feel free to join our chat: https://slack.ory.sh/ 🚀 

Let me know how your journey with Ory goes :)
